### DISCUSSION TITLE
Net Grid usage Power Meter

### DISCUSSION THREAD

**Discussion by oralallen82** [jasonacox/Powerwall-Dashboard#280] [Q&A]
I swear I am missing something. I'd like to add a simple Net Grid usage that basically takes from_grid and subtracts that from to_grid. For the life of me I am unable to accomplish this. Any help would be greatly appreciated.

**Comment by youzer-name** [comment]
This is the opposite, subtracting to_grid from from_grid, but you can just reverse the terms.


![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/57046429/4f7133f2-6e1f-4425-9f42-5d8ae66e906a)

**Reply by oralallen82** [reply]
Thanks for the quick reply. I appreciate your assistance.

**Comment by prussell69** [comment]
I did something a little different so I could mimmick my utility companies meter reading.
I created a variable called "Meter_Offset", which allows me to subtract the difference between the Powerwall-Dashboards calculated value and my utility. Since I started using Powerwall-Dashboard several months before my net meter was installed, I had to do this to correct the display to match my meter reading.
Here's my query:
SELECT ((integral("from_grid")  / 3600000) - (integral("to_grid")  / 3600000) - $Meter_Offset) FROM "autogen"."http"

I don't use the "$timeFilter" since I want the "to_grid" & "from_grid" reading from day-1.
Hope this helps you...
